src/vs/workbench/api/browser/mainThreadWebviewPanels.ts

```diff
@@ -90,6 +90,15 @@ export class MainThreadWebviewPanels extends Disposable implements extHostProtoc
 
 	private readonly webviewOriginStore: ExtensionKeyedWebviewOriginStore;
 
+	public override dispose(): void {
+		for (const webview of this._webviewInputs) {
+			if (typeof webview.group === 'number' && webview.webview.options.autoCloseWhenDispose) {
+				this._editorService.closeEditor({ editor: webview, groupId: webview.group });
```

Comment:
`webview.dispose()` will do the same. But maybe we can do it explicitly here to make it more readable.